test/HHConfiguration.py

```diff
@@ -58,11 +58,11 @@
             jetPtCut = cms.untracked.double(20),
 
             # BTAG INFO
-            # Working points from https://twiki.cern.ch/twiki/bin/viewauth/CMS/BtagRecommendation80XReReco
+            # Working points from https://twiki.cern.ch/twiki/bin/view/CMS/Phase2HGCRecipes#B_tagging
             discr_name =  cms.untracked.string("pfCombinedMVAV2BJetTags"),
```

Comment:
don't you want to move to deepCSV ?

Reply:
Well, I don't know... Are the b-jets rather central ?
> Note that for |eta|<1.5, cMVAv2 should be used, while for |eta|>1.5, DeepCSV is recommended.

Reply:
it should yes, but let's see on your plots :smile: